### PR TITLE
CLN: requested follow-ups

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/indexing.py [hunk 1]
```diff
@@ -189,8 +187,9 @@ def _get_setitem_indexer(self, key):
         if isinstance(key, range):
             return self._convert_range(key, is_setter=True)
 
+        ax_num = self.axis or 0
```
Comment (jreback) [inline pandas/core/indexing.py]: just use axis

Comment (jbrockmendel) [reply pandas/core/indexing.py]: there is already an `axis` in the namespace

Comment (jreback) [reply pandas/core/indexing.py]: then the other one is wrong (and likely should be called ax), consistency is so important here

Comment (jbrockmendel) [reply pandas/core/indexing.py]: changed

**Comment by jreback** [comment]
lgtm. ping on green.

**Comment by jbrockmendel** [comment]
ping

**Comment by jreback** [comment]
thanks!

**Pull request merged**